components/avatars/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Avatars | Horsemaker's UI</title>
+		<link rel="stylesheet" href="../../default.css" />
+		<link rel="stylesheet" href="style.css" />
+		<link rel="stylesheet" href="avatar.css" />
+	</head>
+	<body>
+		<div class="header">
+			<a class="header-link" href="/index.html"
+				><img
+					class="avatar-sm"
+					src="../../assets/horsemaker.svg"
+					alt="🦄"
+				/>
+				<h2>Horsemaker's UI</h2></a
+			>
+		</div>
+		<div class="main avatars">
+			<h1>Avatars</h1>
+			<p>
+				Avatar can be used to show user's profile picture on profile
+				information page, on navigation bar, in blogs grid items.
+			</p>
+			<p>large avatar</p>
+			<div class="playground">
+				<img
+					class="avatar-round avatar-lg"
+					src="../../assets/avatarOne.png"
+					alt=""
```

Comment:
Please add some value in alt in all image tags

Reply:
sure 👍